db/migrate/20211104154340_add_get_tickets_button_label_to_events.rb

```diff
@@ -0,0 +1,5 @@
+class AddGetTicketsButtonLabelToEvents < ActiveRecord::Migration
+  def change
+    add_column :events, :get_tickets_button_label, :string, default: nil
```

Comment:
Do we want to also add the flag to `misc_event_info`?

Reply:
Yes, `misc_event_info` would be the right place.